server.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+
+const app = express();
```

Comment:
It's a good practice to add a middleware to parse incoming JSON payloads. You can use `express.json()` for this. [medium]